src/RoleObjects/__tests__/AccessibilityObject.test.js

```diff
@@ -1,8 +1,8 @@
 import * as createjs from 'createjs-module';
 import ReactTestUtils from 'react-dom/test-utils';
 import KeyCodes from 'keycodes-enum';
-import AccessibilityModule from '../index';
-import { parentEl, stage, container } from '../__jestSharedSetup';
+import AccessibilityModule from '../../index';
+import { parentEl, stage, container } from '../../__tests__/__jestSharedSetup';
```

Comment:
I see that this line is repeated in every test file, I am just thinking aloud like can't we do some configuration in webpack such that it automatically gets imported instead of importing it each file

Reply:
not aware of another way to do that - @mmoss?

Reply:
I'd prefer not to add aliases. IDEs and other tooling handle migrating files more easily when they use relative paths.

Reply:
One option to consider is using https://jestjs.io/docs/configuration#setupfiles-array. With that we can just have jest execute the file, and we can put the objects we want to have access in the global scope (e.g., `global.canvasEl`)